dev-tools/api-request-templates/insomnia_base_opensearch_requests_export.json

```diff
@@ -0,0 +1,838 @@
+		"parentId": "wrk_dd8635e61cac41c3acc82965d35b263e",
+		"modified": 1670269982539,
+		"created": 1666216212269,
+		"url": "{{host}}/movies/_search",
+		"name": "index_documents_all",
+		"description": "",
+		"method": "GET",
+		"body": {
+			"mimeType": "application/json",
+			"text": "{\n    \"query\": {\n        \"match_all\": {}\n    }\n}"
+		},
+		"parameters": [],
+		"headers": [{
+			"name": "Content-Type",
+			"value": "application/json"
+		}],
+		"authentication": {},
+		"metaSortKey": -1666143204817.7578,
+		"isPrivate": false,
+		"settingStoreCookies": true,
+		"settingSendCookies": true,
+		"settingDisableRenderRequestBody": false,
+		"settingEncodeUrl": true,
+		"settingRebuildPath": true,
+		"settingFollowRedirects": "global",
+		"_type": "request"
+	}, {
+		"_id": "req_1a8d787d1db746a9bf803f45e6d55fca",
+		"parentId": "wrk_dd8635e61cac41c3acc82965d35b263e",
+		"modified": 1670269980950,
+		"created": 1667418289544,
+		"url": "{{host}}/movies/_settings",
+		"name": "index_settings",
+		"description": "",
+		"method": "GET",
+		"body": {},
+		"parameters": [],
+		"headers": [],
+		"authentication": {},
+		"metaSortKey": -1666143204817.5625,
+		"isPrivate": false,
+		"settingStoreCookies": true,
+		"settingSendCookies": true,
+		"settingDisableRenderRequestBody": false,
+		"settingEncodeUrl": true,
+		"settingRebuildPath": true,
+		"settingFollowRedirects": "global",
+		"_type": "request"
+	}, {
+		"_id": "req_b2e1b38ecf9f44708c147e823e7cf518",
+		"parentId": "wrk_dd8635e61cac41c3acc82965d35b263e",
+		"modified": 1666144927612,
+		"created": 1666144652556,
+		"url": "{{host}}/_nodes/_all",
+		"name": "nodes_all",
+		"description": "",
+		"method": "GET",
+		"body": {},
+		"parameters": [],
+		"headers": [],
+		"authentication": {},
+		"metaSortKey": -1666143204816,
+		"isPrivate": false,
+		"settingStoreCookies": true,
+		"settingSendCookies": true,
+		"settingDisableRenderRequestBody": false,
+		"settingEncodeUrl": true,
+		"settingRebuildPath": true,
+		"settingFollowRedirects": "global",
+		"_type": "request"
+	}, {
+		"_id": "req_ffeee0c6600948569f94aa86755c226d",
+		"parentId": "wrk_dd8635e61cac41c3acc82965d35b263e",
+		"modified": 1666379764532,
+		"created": 1666379690240,
+		"url": "{{host}}/_nodes/_all,master:true",
+		"name": "nodes_master_eligible",
+		"description": "",
+		"method": "GET",
+		"body": {},
+		"parameters": [],
+		"headers": [],
+		"authentication": {},
+		"metaSortKey": -1666143204791,
+		"isPrivate": false,
+		"settingStoreCookies": true,
+		"settingSendCookies": true,
+		"settingDisableRenderRequestBody": false,
+		"settingEncodeUrl": true,
+		"settingRebuildPath": true,
+		"settingFollowRedirects": "global",
+		"_type": "request"
+	}, {
+		"_id": "req_e67a074c556b4378b373d1d3587609ae",
+		"parentId": "wrk_dd8635e61cac41c3acc82965d35b263e",
+		"modified": 1670269963256,
+		"created": 1666386600349,
+		"url": "{{host}}/_nodes/XSgLo-7wQ-WTOQn2C8P7tg/settings",
```

Comment:
Extremely possible I'm misunderstanding this, but this looks like it's calling a specific node? I assume that's a node id that's unique to your cluster. Intentional for this to be included?

Reply:
You are correct.  This one in particular should be replaced with a placeholder here as this doesn't give much context for new users. There are places though for the sake of getting started quickly I have put sample data such as an index named "movies". I don't have a huge preference in general on say using sample data versus a placeholder, but lmk if you have other thoughts. A nice in-between may be to use sample data such as "sample-index"  instead of "movies" to give more context.